### DISCUSSION TITLE
WebDAV: Manually trigger Nextcloud sync

### DISCUSSION THREAD

**Discussion by meckomorte** [photoprism/photoprism#4109] [General]
I have successfully connected my nextcloud to photoprism using webdav but I cannot get it to sync. I've had it connected before but something happened in Dec that caused it to remove nextcloud. Now this connection its just not synching. I've tried updating to sync every hour, every day etc but it still says never. Is there a way to manually sync? Or do I need to do something else? I've looked through the discussions and I've seen something about sleep settings, would that help if it's never synched? Its been up connected for 3 days.

**Comment by graciousgrey** [comment]
You may get more insight as to where the problem could be if you set the log level to trace mode and observe the logs.

https://docs.photoprism.app/getting-started/troubleshooting/logs/#__tabbed_1_3

Depending on the use case, it may make more sense to [configure Nextcloud's Photos folder as PhotoPrism's originals folder (or a subfolder of it)](https://docs.photoprism.app/getting-started/docker-compose/#volumes) instead of using the [remote sync](https://docs.photoprism.app/user-guide/sync/services-sync/). This way, both apps would work with the same files.

**Reply by meckomorte** [reply]
Ok, so I don't know what setting I did but it finally synced. Thought it's not holding the same folder configuration that nextcloud has. Would that be found in the logs? (I already checked and it's mounted to the right path)

**Reply by graciousgrey** [reply]
So it is working now? 
If you use the [remote sync](https://docs.photoprism.app/user-guide/sync/services-sync/) feature, please make sure that the source and destination folders are not identical, as this can lead to data loss.
